### ISSUE TITLE
Multiple OrderBy support?

### ISSUE THREAD

**Issue by JasonkayZK** [p2panda/aquadoggo#594]
I saw `orderBy` and `orderDirection` in the document.

But it seems that it only support sorting by one field.

So are there any chances to support multiple fields orderBy query?

Thank you.

**Comment by adzialocha** [comment]
> But it seems that it only support sorting by one field.

Yes, that's correct! Currently we do not support multiple `orderBy` fields, also it is not a priority right now but let us know if you have any ideas on how to implement it! We should have the logic in place, question is how one would design a GraphQL API around it?

**Comment by JasonkayZK** [comment]
I have no ideas right now, but once i figured it out, i will let you guys know ASAP. ❤️
